packages/docusaurus-theme-classic/src/theme/SearchBar/index.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, {useState} from 'react';
+
+let Modal: any = null;
```

Comment:
Not very fan of the Algolia solution, for now please just import the modal and we'll lazy load later if needed